docs/developing/information/block-explorers/tzkt.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: TzKT Tezos blockchain explorer
+authors: Baking Bad
+last_update:
+  date: 12 December 2023
+---
+
+import Figure from '@site/src/components/Figure';
+
+## TzKT Tezos blockchain explorer
```

Comment:
This header isn't necessary because docusaurus will automatically use the title value in the front matter as the page title:
![Screenshot 2023-12-13 at 2 47 36 PM](https://github.com/trilitech/tezos-developer-docs/assets/6494785/ca1891c4-6590-4a47-b429-a47d156295ac)

Reply:
Sure. Should i remove it and push once again?

Reply:
Yes, please.

Reply:
Done